packages/material-ui/src/test-utils/createMuiRender.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import * as PropTypes from 'prop-types';
+import { cleanup, render } from 'react-testing-library';
+
+function muiRender(element, options = {}) {
+  const { disableUnnmount = false, strict } = options;
+  const Mode = strict ? React.StrictMode : React.Fragment;
+
+  function TestWrapper({ children }) {
+    return <Mode>{children}</Mode>;
+  }
+  TestWrapper.propTypes = {
+    children: PropTypes.node,
+  };
+
+  if (!disableUnnmount) {
+    cleanup();
+  }
+
+  const result = render(element, { wrapper: TestWrapper });
+
+  /**
+   * convenience helper. Better than repeating all props.
+   */
+  result.setProps = function setProps(props) {
+    result.rerender(React.cloneElement(element, props));
+    return result;
+  };
+
+  return result;
+}
+
+export function createMuiRender(globalOptions = {}) {
```

Comment:
Don't like the name here. It mostly stems from the mount vs. render terminology used in enzyme. Open to bikeshedding.

Reply:
`createClientRender` seems like a good fit considering the current `createRender` uses SSR APIs.